packages/render-fidelity-tools/src/artifact-creator.ts

```diff
@@ -188,7 +198,8 @@ export class ArtifactCreator {
 
   async captureScreenshot(
       renderer: string, scenarioName: string, dimensions: Dimensions,
-      outputPath: string = join(this.outputDirectory, 'model-viewer.png')) {
+      outputPath: string = join(this.outputDirectory, 'model-viewer.png'),
+      useMaxTime: boolean = false, maxTime: number = 10) {
```

Comment:
im just using an arbitrary maxTime here, so feel free to change

Reply:
Looks like it wasn't long enough for sponza. Software rendering can be impressively slow, and keep in mind this also includes things like decoding textures, since it's the first frame. Let's go with 30 or 60 seconds.